waggle-dance-core/src/test/java/com/hotels/bdp/waggledance/mapping/service/impl/MonitoredDatabaseMappingServiceTest.java

```diff
@@ -95,6 +96,11 @@ public void onUpdate() throws Exception {
     verify(wrapped).onUpdate(metaStore, newMetastore);
   }
 
+  @Test
+  public void getWrappedService() {
+    assertThat(service.getWrappedService(), is((DatabaseMappingService) wrapped));
```

Comment:
I think this method `getWrappedService` is actually not used anywhere, can we delete it (then this test can also go)